packages/ketcher-polymer-editor-react/src/styles/theme.js

```diff
@@ -24,7 +24,8 @@ const color = {
     primary: '#121212',
     secondary: '#BCBCBC',
     light: '#FFFFFF',
-    dark: '#000000'
+    dark: '#5B6077',
+    black: '#000000'
```

Comment:
I don't think we should extend our theme schema with `black` color, it seems to defeat the purpose of a theme.

If this exact color is needed (and won't change if color palette changes) maybe let's just use it without a theme?